Formula/grokj2k.rb

```diff
@@ -20,19 +20,49 @@ class Grokj2k < Formula
 
   depends_on "cmake" => :build
   depends_on "doxygen" => :build
-  depends_on "exiftool"
```

Comment:
Went with trying the easier option of dropping Perl and ExifTool.

Due to how Homebrew Perl formulae are installed, this makes enabling ExifTool a bit more complicated. Since I am not familiar with how to test this works, I didn't pursue this path.

As reference, I think some additional install logic like following would be necessary to get CMake to detect Perl lib on macOS, detect ExifTool libraries, add RPATH, and make sure ExifTool libraries are detectable in final binaries:
```rb
    ENV.prepend_create_path "PERL5LIB", Formula["exiftool"].opt_libexec/"lib"

    cmake_args = std_cmake_args + ["-DBUILD_DOC=ON"]
    on_macos do
      perl_version = Utils.safe_popen_read("perl", "--version")[/v(\d+\.\d+)(?:\.\d+)?/, 1]
      perl_core = MacOS.sdk_path/"System/Library/Perl"/perl_version/"darwin-thread-multi-2level/CORE"
      cmake_args << "-DPERL_INCLUDE_PATH=#{perl_core}"
    end
    on_linux do
      perl_core = Formula["perl"].opt_lib/"perl5"/Formula["perl"].version/"x86_64-linux-thread-multi/CORE"
      ENV.append "LDFLAGS", "-Wl,-rpath=#{perl_core}"
    end

    mkdir "build" do
      system "cmake", "..", *cmake_args
      system "make", "install"
      include.install_symlink "grok-#{version.major_minor}" => "grok"
    end

    bin.env_script_all_files libexec/"bin", PERL5LIB: ENV["PERL5LIB"]
```